<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="resultImgLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>431</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="minValueSlider">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>22</width>
      <height>601</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>620</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="chooseFileBtn"/>
   <addaction name="separator"/>
   <addaction name="tightenImgBtn"/>
   <addaction name="separator"/>
   <addaction name="equalizeImgBtn"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="chooseFileBtn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/dir.png</normaloff>:/images/icons/dir.png</iconset>
   </property>
   <property name="text">
    <string>Choose file</string>
   </property>
   <property name="toolTip">
    <string>Choose file</string>
   </property>
  </action>
  <action name="tightenImgBtn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/wrench.png</normaloff>:/images/icons/wrench.png</iconset>
   </property>
   <property name="text">
    <string>Tighten image</string>
   </property>
   <property name="toolTip">
    <string>Tighten image</string>
   </property>
  </action>
  <action name="equalizeImgBtn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/equalizer.png</normaloff>:/images/icons/equalizer.png</iconset>
   </property>
   <property name="text">
    <string>Equalize image</string>
   </property>
   <property name="toolTip">
    <string>Equalize image</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>minValueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>654</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>675</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
